pyscriptjs/src/main.ts

```diff
@@ -418,6 +418,8 @@ modules must contain a "plugin" attribute. For more information check the plugin
         this.incrementNumPendingTags();
         this.decrementNumPendingTags();
         await this.scriptTagsPromise;
+        // todo: don't pollute namespace...
+        this.interpreter.run('import asyncio; asyncio.get_event_loop().start_()');
```

Comment:
what is this line doing? Why it was not needed before?

Reply:
We have a custom event loop that delays tasks until we call this `start_` function. We have tests like `test_async_order` that assert that all Pyscript tags are executed before any async code. Since now we have a bunch of async breakpoints everywhere that won't happen unless we're explicit about it.

Reply:
got it. Then I suppose I would like the API to be something like `import pyscript; pyscript._start_event_loop()` or something similar, but I'm happy with this for now and move it to another API later

Reply:
I am working on bringing #1262 up to - date with esbuild. We should merge that first and split `pyscript/__init__.py` up into several files and that will help a lot with code organization

Reply:
I agree, but let's do it after we merge this PR. I would like this to happen ASAP because it's the bottleneck for starting to experiment with workers

Reply:
this is reverted as a part of getting this PR into a mergeable state.

Reply:
resolving...